app-scroll-effects/effects/resize-title.html

```diff
@@ -25,8 +25,8 @@
     Polymer.AppLayout.registerEffect('resize-title', {
       /** @this Polymer.AppLayout.ElementWithBackground */
       setUp: function setUp() {
-        var title = Polymer.dom(this).querySelector('[title]');
-        var condensedTitle = Polymer.dom(this).querySelector('[condensed-title]');
+        var title = this._getDOMRef('title');
+        var condensedTitle = this._getDOMRef('condensedTitle');
```

Comment:
This will create `_bgContainer` and I don't think is not needed for resize-title effect.